src/Repository/TriggerRepository.php

```diff
@@ -155,6 +155,10 @@ public static function registerWpTriggers()
 			Register::trigger(new Trigger\WordPress\UpdatesAvailable());
 		}
 
+//		if (getSetting('triggers/wordpress/email_address_changed')) {
+			Register::trigger(new Trigger\WordPress\EmailChanged());
+//		}
+
```

Comment:
I commented the code here because it should probably be added here, but these options from these functions doesn't work yet?

Reply:
What do you mean it doesn't work yet? What are the symptoms?

Reply:
I checked it more thoroughly. And here all the settings just don't save in Settings -> Triggers -> WordPress. It's probably a bug that it doesn't work here.

Reply:
Sure. Do it in this branch or in the new one?

Reply:
Ideally on a new one, then we'll merge that fix, and then this PR

Reply:
As said in the previous comment, we don't need that option here, so just register that trigger without any conditions